pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	cadvisorfs "github.com/google/cadvisor/fs"
 	cadvisorapiv2 "github.com/google/cadvisor/info/v2"
 	"github.com/stretchr/testify/assert"
+	assertModule "github.com/stretchr/testify/assert"
```

Comment:
looks like we already import this.  Also, `assertModule` is camelcased, which imports shouldn't be.

Reply:
+1 on name. The reason it is renamed is that we declare variable with the same name `assert` and cannot use `assert.ObjectsAreEqual` method after this - variable overrides the package name